recipes/pytest-raises/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pytest-raises" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4710a8921b0b7425cd2073d696862ecf99922ef522e681cc63a52008112d63d9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytest >=2.8.1
```

According to the ``setup.py`` it should be
```suggestion
    - pytest >=3.2.2
```

---

https://github.com/Lemmons/pytest-raises/blob/d0b31589acfffd1bf5c562a4c5adbaa157972696/setup.py#L108